src/components/Votes.jsx

```diff
@@ -1,9 +1,43 @@
-export default function Votes() {
+import { useState } from "react";
+import { patchVotes } from "../api";
+
+export default function Votes({ votes, article_id }) {
+  const [localVotes, setLocalVotes] = useState(0);
+  const [stateError, setStateError] = useState(false);
+
+  const handleLike = () => {
+    setLocalVotes((currentVotes) => currentVotes + 1);
+    setStateError(false);
+    patchVotes(article_id, 1).catch((err) => {
+      setStateError(true);
+      setLocalVotes((currentVotes) => currentVotes - 1);
+    });
+  };
```

Comment:
Great implementation of optimistic rendering for the votes ✅ 
Awesome that you've also implemented error handling to inform the user if their vote has been unsuccessful, and revert the vote in state 👍